docs/getting-started/user-roadmap-apiml.md

```diff
@@ -0,0 +1,165 @@
+# Information roadmap for Zowe API Mediation Layer
+
+This roadmap outlines information resources that are applicable to the various user roles who are interested in Zowe API Mediation Layer. These resources provide information about various subject areas, such as learning basic skills, installation, developing, and troubleshooting for Zowe API Mediation Layer.
+
+The following definition of skill levels about Zowe assist you with gathering the most relevant resources for you. 
+
+* Beginner: You're starting out and want to learn the fundamentals.
+* Intermediate: You have some experience but want to learn more in-depth skills. 
```

Comment:
I don't really understand how do I distinguish whether I am intermediate? 
Is it by contributing?
By onboarding service?
By using API ML as a user via catalog?